### ISSUE TITLE
android 3.7.2 broken after android 4.0 fix

### ISSUE THREAD

**Issue by spyro92** [phonegap/phonegap-mobile-accessibility#20]
This is the error I'm getting on android 3.7.2.
```
/Development/mobile-client/platforms/android/src/com/phonegap/plugin/mobileaccessibility/MobileAccessibility.java:140: error: cannot find symbol
                    view = (WebView) webView.getView();
                                            ^
  symbol:   method getView()
  location: variable webView of type CordovaWebView
/Development/mobile-client/platforms/android/src/com/phonegap/plugin/mobileaccessibility/DonutMobileAccessibilityHelper.java:44: error: cannot find symbol
          mWebView = (WebView) mobileAccessibility.webView.getView();
                                                          ^
  symbol:   method getView()
  location: variable webView of type CordovaWebView
/Development/mobile-client/platforms/android/src/com/phonegap/plugin/mobileaccessibility/JellyBeanMobileAccessibilityHelper.java:42: error: cannot find symbol
            view = (WebView) mobileAccessibility.webView.getView();
                                                        ^
  symbol:   method getView()
  location: variable webView of type CordovaWebView
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
3 errors
```

Rolling back the last 2 commits fixes the issue for me

To this point:
https://github.com/phonegap/phonegap-mobile-accessibility/tree/2fb3aa4c2b4600f6a845631902152fdde04d40f0

**Comment by jernejc** [comment]
same issue

**Comment by majornista** [comment]
Should be corrected by: 415b357e9a24e971df0ccee7d0413f30ab3ab1d7.
